python/snewpy/models/presn_loaders.py

```diff
@@ -139,3 +139,36 @@ def get_initial_spectra(self, t, E, flavors=Flavor):
         flux = self.interpolated(t, E) / (u.MeV * u.s)
         return {f: flux[f] for f in flavors}
 
+class Yoshida_2016(SupernovaModel):
+    """Set up a presupernova model based on 
+    [Yoshida et al. (2016), PRD 93, 123012.]
+    """
+    def __init__(self, filename):
+        with open(self.request_file(filename)) as f:
+            data = []
+            T = []
+            while (line := f.readline()):
+                if(not line):break
+                N_step,T_step,logT_c,logRho_c = [conv(token) for conv,token in zip([int,float,float,float],line.split())]
+                data += [[np.loadtxt(f, max_rows=100).flatten() for i in range(4)]]
+                T+=[T_step]
```

Comment:
```suggestion
                if not line: break
                T += [float(line.split()[1])]
                data += [[np.loadtxt(f, max_rows=100).flatten() for i in range(4)]]
```

Reply:
This adds a "magic number" `[1]` here, so for me it's less clear to see, what is going on.
But it's subjective. So if it makes code look better to you, let's do it